enclave/core/sgx/globals.c

```diff
@@ -189,9 +190,29 @@ size_t __oe_get_reloc_size()
 **==============================================================================
 */
 
-const void* __oe_get_eeid()
+const oe_eeid_page_t* __oe_get_eeid_page()
 {
-    return oe_eeid;
+    if (!_eeid_rva)
+        return NULL;
+
+    const unsigned char* base = __oe_get_enclave_base();
+    return (oe_eeid_page_t*)(base + _eeid_rva);
+}
+
+const oe_eeid_t* __oe_get_eeid()
+{
+    if (!_eeid_rva)
+        return NULL;
+
+    const oe_eeid_page_t* eeid_page = __oe_get_eeid_page();
+    return eeid_page && eeid_page->eeid.version == OE_EEID_VERSION
+               ? &eeid_page->eeid
+               : NULL;
+}
+
+const oe_enclave_config_t* __oe_get_enclave_config()
```

Comment:
This seems to be another reason to carry the `config_id` within `oe_eeid_t` ... as far as I can tell, it's only used by the attester implementation:
- the attester plugin already has to `__oe_get_eeid()` today, so it makes sense for `config_id` to accompany it instead of carrying it as a new global.
- on KSS systems, it doesn't need to inject the custom claim, the base evidence has it already, so it also doesn't have a use for `__oe_get_enclave_config()`

Reply:
Yep, that's where the `config_id` is now.